src/_guides/language/language-tour.md

```diff
@@ -4520,7 +4520,7 @@ Documentation comments are multi-line or single-line comments that begin
 with `///` or `/**`. Using `///` on consecutive lines has the same
 effect as a multi-line doc comment.
 
-Inside a documentation comment, the Dart compiler ignores all text
+Inside a documentation comment, Dart progam analysis ignores all text
```

Comment:
Specific to this change, I think the alternate that you suggested of using 'the analyzer' might read better than 'Dart program analysis'. I think enough users have an understanding of what the analyzer is that it won't be confusing. Either way it's better than 'compiler' because all of the compilers ignore doc comments.

@kwalrath I don't think I've ever looked at this part of the documentation before. I don't know whether it matters much to users, but I wanted to point out that the analyzer and dartdoc treat the text slightly differently. As stated on line 4526, the analyzer will only resolve names in brackets if they're in the lexical scope (that is, if there's an import for them) whereas dartdoc will do extra work and create links for things even if they haven't been imported. Analyzer will ignore identifiers in brackets if they aren't in the lexical scope.

Also, a minor nit, but on line 4541, it's idiomatic to use the name of a parameter rather than the type of the parameter when documenting a method, so consider changing `[Food]` to `[food]`. (Using the type is confusing when there are multiple parameters with the same type.)

Reply:
Thanks for the suggestions @bwilkerson, I've gone ahead and switched the wording to `the analyzer` which I now noticed is used elsewhere in the page anyway. I've also updated the parameter references to `[food]`. 

As for the dartdoc vs analyzer lexical scope issue, I tried to modify the piece to provide more explanation but I couldn't find a great way to word it. If it is desired to provide more specifics about those differences, perhaps it would be better handled by @kwalrath as a commit to this branch or a separate PR.